providers/openmetrics/client_test.go

```diff
@@ -33,12 +29,13 @@ type ClientInterceptorTestSuite struct {
 	testClient     pb_testproto.TestServiceClient
 	ctx            context.Context
 	cancel         context.CancelFunc
+	clientMetrics  *ClientMetrics
 }
 
 func (s *ClientInterceptorTestSuite) SetupSuite() {
 	var err error
 
-	DefaultClientMetrics.EnableClientHandlingTimeHistogram()
+	s.clientMetrics = NewClientMetrics(WithClientHandlingTimeHistogram())
```

Comment:
We don't even need to register this with a registry now - we test how interceptors increment counters/etc, don't care about the registry.